src/components/grid/Grid.stories.tsx

```diff
@@ -37,6 +61,76 @@ export const defaultContainer = (): React.ReactElement => (
   </GridContainer>
 )
 
+export const customElements = (): React.ReactElement => (
+  <GridContainer<CustomGridContainerProps> asCustom={CustomGridContainer}>
+    <li>
+      <Grid<CustomGridProps> asCustom={CustomGrid}>
+        <Grid col={11}>{testContent}</Grid>
+        <Grid col={2}>{testContent}</Grid>
+      </Grid>
+    </li>
```

Comment:
Now that you've updated your custom grid to be a `li`, this is going to render `li` nested within a `li`. Does it work you you get rid of the parent `li` tags like so?:

```suggestion
    <Grid<CustomGridProps> asCustom={CustomGrid}>
      <Grid col={11}>{testContent}</Grid>
      <Grid col={2}>{testContent}</Grid>
    </Grid>
```